oioioi/contests/templates/contests/select_contest.html

```diff
@@ -8,7 +8,31 @@
 <article>
     <h1>{% trans "Select contest" %}</h1>
 
-    <div class="table-responsive-md">
+    <div style="width: 20%; margin-bottom: 1rem;">
+            <form method="GET" action="{% url 'filter_contests' filter_value='PLACEHOLDER' %}" id="filter_form">
+                {% csrf_token %}
+                <div class="input-group">
+                <input type="text" id="filter_input" class="form-control search-query" style="width: 20%;" placeholder="Search" name="filter_field" value="{{ filter }}">
+                <span class="input-group-btn">
+                    <button type="submit" class="btn btn-outline-secondary "  name="submit_button"> <svg class="svg-inline--fa fa-magnifying-glass" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="magnifying-glass" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" data-fa-i2svg=""><path fill="currentColor" d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352a144 144 0 1 0 0-288 144 144 0 1 0 0 288z"></path></svg> </button>
```

Comment:
there is no need to add svg tag, 
`<i class="fa-solid fa-magnifying-glass"></i>`
should be enough

Reply:
fixed